lib/ipmi_dcmi.c

```diff
@@ -2056,1485 +1766,123 @@ ipmi_dcmi_pwr_actdeact(struct ipmi_intf * intf, uint8_t option)
 }
 /* end power management activate/deactivate */
 
-/* Node Manager discover */
-static int
-_ipmi_nm_discover(struct ipmi_intf * intf, struct nm_discover *disc)
-{
-	struct ipmi_rq req; /* request data to send to the BMC */
-	struct ipmi_rs *rsp;
-	uint8_t msg_data[3]; /* 'raw' data to be sent to the BMC */
-
-	msg_data[0] = 0x57;
-	msg_data[1] = 1;
-	msg_data[2] = 0;
-	memset(&req, 0, sizeof(req));
-	req.msg.netfn = IPMI_NETFN_OEM;
-	req.msg.cmd = IPMI_NM_GET_VERSION;
-	req.msg.data = msg_data;
-	req.msg.data_len = 3;
-	rsp = intf->sendrecv(intf, &req);
-	if (chk_nm_rsp(rsp)) {
-		return -1;
-	}
-	memcpy(disc, rsp->data, sizeof (struct nm_discover));
-	return 0;
-}
-/* Get NM capabilities
- *
- * This function returns the available capabilities of the platform.
- *
- * returns success/failure
- *
- * @intf:   ipmi interface handler
- * @caps:   fills in capability struct
- */
 static int
-_ipmi_nm_getcapabilities(struct ipmi_intf * intf, uint8_t domain,
-                         uint8_t trigger, struct nm_capability *caps)
+ipmi_dcmi_set_limit(struct ipmi_intf * intf, int argc, char **argv)
 {
-	struct ipmi_rq req; /* request data to send to the BMC */
-	struct ipmi_rs *rsp;
-	uint8_t msg_data[5]; /* 'raw' data to be sent to the BMC */
+	int rc = 0;
 
-	msg_data[0] = 0x57;
-	msg_data[1] = 1;
-	msg_data[2] = 0;
-	msg_data[3] = domain;
-	msg_data[4] = trigger; /* power control policy or trigger */
-	memset(&req, 0, sizeof(req));
-	req.msg.netfn = IPMI_NETFN_OEM;
-	req.msg.cmd = IPMI_NM_GET_CAP;
-	req.msg.data = msg_data;
-	req.msg.data_len = 5;
-	rsp = intf->sendrecv(intf, &req);
-	if (chk_nm_rsp(rsp)) {
-		return -1;
-	}
-	memcpy(caps, rsp->data, sizeof (struct nm_capability));
-	return 0;
-}
+	if ( argc == 10) {
```

Comment:
nit: extra whitespace

Reply:
I see now this is just moved code, so perhaps it should be left as-is in that case.